CodingPractice/PythonAssignments/tests/cleancode/test_CountItemsInLIst.py

```diff
@@ -9,7 +9,7 @@ def test_return_zero_for_empty_list(self):
         self.assertEqual(0, count_items_in_list([]))
 
     def test_non_empty_list(self):
-        self.assertEqual(5, count_items_in_list([0, 1, 2, 3, 4, 5]))
+        self.assertEqual(6, count_items_in_list([0, 1, 2, 3, 4, 5]))
```

Comment:
Well done you correctly identified that there are 6 items in the list.
I deliberately made my list start at zero - a typical oversight.